packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -74,7 +74,10 @@ class CreateAccount extends Component {
 
     render() {
         const { errorMessage, handleSubmit, initialValues, props } = this;
-        const { classes } = props;
+        const { classes, isSignedIn } = props;
+        if (isSignedIn) {
```

Comment:
I wouldn't say no to a blank line between the destructuring and the conditional here, either 😄 , will help with readability.